backend/src/schema/resolvers/user_management.spec.js

```diff
@@ -311,121 +310,3 @@ describe('change password', () => {
-    const variables = {
-      id: 'bcb2d923-f3af-479e-9f00-61b12e864667',
-      password: 'xYz',
-      slug: 'apfel-strudel',
-      name: 'Apfel Strudel',
-      email: 'apfel-strudel@test.org',
-    }
-    await authenticatedClient.request(
-      gql`
-        mutation($id: ID, $password: String!, $slug: String, $name: String, $email: String!) {
-          CreateUser(id: $id, password: $password, slug: $slug, name: $name, email: $email) {
-            id
-          }
-        }
-      `,
-      variables,
-    )
-  }
-
-  beforeEach(async () => {
-    const adminParams = {
-      role: 'admin',
-      email: 'admin@example.org',
-      password: '1234',
-    }
-    // create an admin user who has enough permissions to create other users
-    await factory.create('User', adminParams)
-    const headers = await login(adminParams)
-    authenticatedClient = new GraphQLClient(host, { headers })
-    // but also create an unauthenticated client to issue the `User` query
-    client = new GraphQLClient(host)
-  })
-
-  describe('unauthenticated query of "publicKey" (does the RSA key pair get generated at all?)', () => {
-    it('returns publicKey', async () => {
-      await generateUserWithKeys(authenticatedClient)
-      await expect(
-        await client.request(queryUserPuplicKey, { queriedUserSlug: 'apfel-strudel' }),
-      ).toEqual(
-        expect.objectContaining({
-          User: [
-            {
-              id: 'bcb2d923-f3af-479e-9f00-61b12e864667',
-              publicKey: expect.any(String),
-            },
-          ],
-        }),
-      )
-    })
-  })
-
-  describe('unauthenticated query of "privateKey"', () => {
-    it('throws "Not Authorised!"', async () => {
-      await generateUserWithKeys(authenticatedClient)
-      await expect(
-        client.request(queryUserPrivateKey, { queriedUserSlug: 'apfel-strudel' }),
-      ).rejects.toThrow('Not Authorised')
-    })
-  })
-
-  // authenticate
-  beforeEach(async () => {
-    headers = await login({ email: 'test@example.org', password: '1234' })
-    client = new GraphQLClient(host, { headers })
-  })
-
-  describe('authenticated query of "publicKey"', () => {
-    it('returns publicKey', async () => {
-      await generateUserWithKeys(authenticatedClient)
-      await expect(
-        await client.request(queryUserPuplicKey, { queriedUserSlug: 'apfel-strudel' }),
-      ).toEqual(
-        expect.objectContaining({
-          User: [
-            {
-              id: 'bcb2d923-f3af-479e-9f00-61b12e864667',
-              publicKey: expect.any(String),
-            },
-          ],
-        }),
-      )
-    })
-  })
-
-  describe('authenticated query of "privateKey"', () => {
-    it('throws "Not Authorised!"', async () => {
-      await generateUserWithKeys(authenticatedClient)
-      await expect(
-        client.request(queryUserPrivateKey, { queriedUserSlug: 'apfel-strudel' }),
-      ).rejects.toThrow('Not Authorised')
-    })
-  })
-})
```

Comment:
All my sweat and blood code gets deleted 😱😢😂

Reply:
I thought the same hehe
but from a safety perspective, a big improvement :smile_cat: